docs/gallery/plot_convert_nwb_hdf5.py

```diff
@@ -60,6 +60,8 @@
 with NWBHDF5IO(filename, 'r', load_namespaces=False) as read_io:  # Create HDF5 IO object for read
     with NWBZarrIO(zarr_filename, mode='w') as export_io:         # Create Zarr IO object for write
         export_io.export(src_io=read_io, write_args=dict(link_data=False))   # Export from HDF5 to Zarr
+        export_io.close()
+    read_io.close()
```

Comment:
We should not need to call ``close`` on the IO objects explicitly.  ``close`` is being called by the ``__exit__`` function here:

https://github.com/hdmf-dev/hdmf/blob/23a8a2638f060366ec1e7789f1344c9732e42e43/src/hdmf/backends/io.py#L133-L134

when we leave the context

Reply:
I removed extraneous io.close calls in #61